assets/scripts/keyboard.js

```diff
@@ -379,7 +379,7 @@ document.getElementById("keyboard-cont").addEventListener("click", (e) => {
 document.addEventListener("keyup", (e) => {
 
    let pressedKey = String(e.key)
-   let found = pressedKey.match(/[a-z]/gi)
+   let found = pressedKey.match(/^[a-z]$/gi)
```

Comment:
I'm wondering if this is the best way to check this though. Regex are relatively costly and slow for something like this, wouldn't it be more efficient just check that the input is within the range a to z, which causes JS to compare the ascii key code? i.e. `pressedKey >= 'a' && pressedKey <= 'z'`.

Reply:
no, not really. regex isnt that costly. if we really needed to change it than something like
```js
pressedKey.toUpperCase() != char.toLowerCase() || pressedKey.codePointAt(0) > 127
```